app/js/components/orderEntry/addForm/AddForm.jsx

```diff
@@ -217,19 +217,22 @@ export class AddForm extends React.Component {
 
     if (Object.values(this.state.fieldErrors).includes(true)) return true;
```

Comment:
Have you tried `return Object.values(this.state.fieldErrors).includes(true)`?
I think it will give us the same result

Reply:
If I use the return statement there, all the other code below will not be reachable.

Reply:
Well, I think your code will also make the lines after it unreachable when the condition is true. return statements generally block the execution of the statements after them

Reply:
@larrystone With the way this is implemented it is working okay because of the condition `if` but using what you have suggested makes the code unreachable because the condition is removed.